ui/src/app/kvm/views/KVMDetails/KVMSummary/KVMNumaResources/_index.scss

```diff
@@ -0,0 +1,10 @@
+@mixin KVMNumaResources {
+  .numa-resources-grid {
+    @extend %base-grid;
+    grid-template-columns: 100%;
+
+    @media only screen and (min-width: $breakpoint-x-large) {
```

Comment:
1300px seem like a very high threshold for going from 3 cards to 1. This might not be what the spec says, but could it drop to two columns and then down to one?

Reply:
Yeah I think you're right on this one. It wasn't in the spec but it does make for a pretty enormous card. I'll change it so it's 1300+ = 3 cards, 1030 - 1300 = 2 cards, <1030 = 1 card.